oak/server/rust/oak_glue/src/lib.rs

```diff
@@ -27,9 +27,11 @@ use std::{
 };
 
 #[no_mangle]
-pub extern "C" fn glue_init() {
+pub extern "C" fn glue_init(debug: u32) {
```

Comment:
I actually don't think this should be here, we should probably have a corresponding flag / feature gate in the Rust code instead, which affects the compilation step.